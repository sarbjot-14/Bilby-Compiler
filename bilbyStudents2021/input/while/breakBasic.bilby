main{
	mut count := 0;
	mut count2 := 0;
	while (count < 5) {
		print "outer" $s count $n;
		
		if(count ==3){
				print "count is" $s count $n;
				break;
		}	
		count := count +1;
	
		
	}
}