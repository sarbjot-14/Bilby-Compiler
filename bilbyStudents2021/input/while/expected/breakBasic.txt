outer 0
outer 1
outer 2
outer 3
count is 3
